clang/lib/Driver/ToolChains/Flang.cpp

```diff
@@ -787,6 +787,9 @@ void Flang::ConstructJob(Compilation &C, const JobAction &JA,
 
       if (Args.hasArg(options::OPT_fopenmp_force_usm))
         CmdArgs.push_back("-fopenmp-force-usm");
+      // TODO: OpenMP support isn't "done" yet, so for now we warn that it
+      // is experimental.
+      D.Diag(diag::warn_openmp_experimental);
```

Comment:
Please could you add a test that this warning is printed. `flang/test/Driver/fopoenmp.f90` would be a good place. There is an example in the file already checking for a warning.

Reply:
While I was slightly hesitant to add a test for a temporary warning, I've done so. If anyone else thinks it doesn't belong I'll undo it.